modules/cluster-logging-deploy-cli.adoc

```diff
@@ -342,23 +342,31 @@ spec:
         maxAge: 7d
       audit:
         maxAge: 7d
-    elasticsearch:
       nodeCount: 3 <5>
       storage:
         storageClassName: "<storage-class-name>" <6>
         size: 200G
+      resources: <7>
+        requests:
+          cpu: "1"
+          memory: "16Gi"
```

Comment:
in the `cluster-logging-about-crd.adoc` this is `8Gi`, can we update this value to match that?